Services/data/devicemodels/scripts/EmergencyValveRelease-method.js

```diff
@@ -2,32 +2,38 @@
 
 /*global log*/
 /*global updateState*/
+/*global reportMethodProgress*/
 /*global sleep*/
+/*global disableSensorSimulation*/
+/*global enableSensorSimulation*/
+/*global enableTelemetry*/
+/*global disableTelemetry*/
 /*jslint node: true*/
 
 "use strict";
 
-// Default state
-var state = {
-    pressure: 250.0,
-    CalculateRandomizedTelemetry: false
-};
-
 /**
  * Entry point function called by the simulation engine.
  *
  * @param context        The context contains current time, device model and id
  * @param previousState  The device state since the last iteration
  */
-
 /*jslint unparam: true*/
 function main(context, previousState) {
 
-    // Reboot - devices goes offline and comes online after 20 seconds
-    log("Executing DecreasePressure simulation function.");
+    log("Starting 'Emergency Valve Release' method simulation");
+
+    var state = {
+        simulation_state: "normal_pressure"
+    };
+
+    disableSensorSimulation();
+
     state.pressure = 150;
-    state.CalculateRandomizedTelemetry = true;
-    // update the state to 150
     updateState(state);
+    log("Pressure decreased to " + state.pressure);
+
+    enableSensorSimulation();
```

Comment:
I don't think you want to enable it again as the simulation calculation will overwrite the pressure value you just set.  I think you want to keep it disabled.